packages/amplify-cli/src/amplify-exception-handler.ts

```diff
@@ -202,6 +205,54 @@ const nodeErrorToAmplifyException = (err: NodeJS.ErrnoException): AmplifyExcepti
   );
 };
 
+type CommonExceptionEntry = {
+  messageIndicator: RegExp;
+  amplifyErrorType: AmplifyErrorType;
+  newMessage?: string;
+  resolution: string;
+};
+
+const commonExceptions: CommonExceptionEntry[] = [
+  {
+    messageIndicator: /The provided token has expired./,
+    amplifyErrorType: 'TokenError',
+    newMessage: 'Your AWS access tokens have expired.',
+    resolution: 'Please refresh your tokens.',
```

Comment:
Nit: remove the "please" from all of these resolutions. For the refresh tokens error, maybe change to "Refresh your AWS credentials and retry the operation"